src/main/java/de/zalando/aruha/nakadi/repository/kafka/KafkaTopicRepository.java

```diff
@@ -210,7 +225,8 @@ public TopicPartition getPartition(final String topicId, final String partition)
             topicPartition.setOldestAvailableOffset(toNakadiOffset(consumer.position(tp)));
```

Comment:
Shouldn't particularly the oldest available offset expose BEGIN in case the partition is empty?
Because that makes it transparent to the client and a new / boostrapping / recovering client can just use for reading whatever is exposed in this field.